doc/dev/installation.rst

```diff
@@ -10,6 +10,14 @@ pre-built binaries are being distributed via the Python Package Index (PyPI) for
 
     pip install pennylane-catalyst
 
+.. warning::
+
+    macOS does not ship with a system compiler by default, which Catalyst depends on. Please
+    ensure that `XCode <https://developer.apple.com/xcode/resources/`_ or the
+    ``XCode Command Line Tools`` are installed on your system before using Catalyst.
```

Comment:
Have you tried testing Catalyst with the clang installation from Homebrew? This may be an alternative option here.

Reply:
Haven't tried it but it should be possible. However for regular users installing Xcode is one less step to get Catalyst (no need to install brew).